list_to_email_converter.py

```diff
@@ -0,0 +1,234 @@
+import os
+from urllib.parse import quote
+import sys
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("path",type=str, help="the folderpath like: ./example/more/pytr")
+parser.add_argument("-s","--seek", help="seeks an existing entry", action="store_true")
```

Comment:
I think you don't need this flag: Just create a list of md-files at the beginning. If `path` is a directory (you can test this!), then fill this list with all md-files you find in the path. If `path` is an md-file, the list consist solely of this single file. You can even add the possibility to provide  a list of md-files as input (e.g. using regular expressions like `*_project.md`.